services/worker-manager/src/api.js

```diff
@@ -838,3 +838,22 @@ builder.declare({
 
   return res.reply({ expires: expires.toJSON(), credentials, secret: newSecret });
 });
+
+builder.declare({
+  method: 'get',
+  route: '/__heartbeat__',
```

Comment:
I think it would be good to add either unit-tests for those calls, or include those in smoke test. 
WDYT?

Reply:
Yep, I like that. Good call, thanks!